.yamato/ruamel/jobs/projects/yml_project.py

```diff
@@ -12,12 +13,27 @@ def create_project_ymls(metafile):
     # project_all yml file
     yml = {}
     for editor in metafile['editors']:
-        job = Project_AllJob(metafile["project"]["name"], editor, metafile["all"]["dependencies"])
+        expression = ""
+        if metafile["expression_trigger"]["expression"] != "":
```

Comment:
This is a small thing so no worries if stays as is, but could 16-18 also be skipped, because would it evaluate to whatever the value is anyways?